.github/workflows/maven.yml

```diff
@@ -0,0 +1,27 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Java CI with Maven
```

Comment:
Note that for a badge of the build status, this will become what's to the left of the status (i.e., what is currently "Build").

Speaking of which, I don't see the build status badge change here.

Reply:
I forgot about the badge! I renamed the workflow to "Build with Maven" and added the badge here: https://github.com/phyloref/jphyloref/blob/61381356a18547f964085d5261b049b1cfcce2a3/README.md